### PR TITLE
feat(crio): enabled privileged mode for CRIO in container kill experiment

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ksatchit** [comment]
Lets use Go helper & rename the pod

File: chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml [hunk 1]
```diff
@@ -1,173 +1,148 @@
...
-      register: chaos_service_account
-
-    - name: Patch the chaoslib image
-      template:
-        src:  /chaoslib/litmus/container_kill/containerd_chaos/containerd.j2
-        dest: /tmp/containerd-chaos-pod.yml
-      vars:
-        containerd_image: "{{ lib_image }}"
-        c_svc_acc: "{{ chaos_service_account.stdout }}"
+  - name: Select the app pod
```
Comment (uditgaurav) [inline chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml:3]: Can we follow the banner name here and all places? like [Prepare]

File: chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml [hunk 2]
```diff
@@ -1,173 +1,148 @@
...
+    args:
+      executable: /bin/bash
+    register: app_pod_name
+
+  - name: Record app pod name
+    set_fact:
+      app_pod: "{{ app_pod_name.stdout }}"
+  when: "app_pod is not defined or app_pod == ''"
+
+- name: Identify the application node
```
Comment (uditgaurav) [inline chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml:17]: Same as above "[Prepare: Identify the application node]"

File: chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml [hunk 3]
```diff
@@ -1,173 +1,159 @@
...
-
-    # chaosDiffTimeStamp contains the difference of current timestamp and start timestamp
-    # it will helpful to track the run duration of helper pod
-    - set_fact:
-        chaosDiffTimeStamp: "{{ ((chaosCurrentTimeStamp | to_datetime) - (chaosStartTimeStamp | to_datetime)).total_seconds()  }}"
-
-    - fail:
-        msg: "containerd-chaos pod failed"
-      when: "chaosDiffTimeStamp|int < c_duration|int"
+# Containerd chaos pods creation is attempted for a total of 3 times, if it is not immediately schedulable due to transient node conditions
```
Comment (ksatchit) [inline chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml:90]: the comment here and elsewhere can be changed to reflect new name of the helper pod

**Pull request merged**